package.json

```diff
@@ -71,9 +72,6 @@
     "node": "14.19.1"
   },
   "dependencies": {
-    "@types/ungap__structured-clone": "^0.3.0",
-    "@ungap/structured-clone": "^0.3.4",
-    "i": "^0.3.7",
-    "npm": "^8.7.0"
```

Comment:
I have a hypothesis about how the `npm` and `i` deps may have been introduced:
1. Developer has intent to install new packages @ungap/structured-clone and @types/ungap__structured-clone
2. Developer starts typing into the terminal `npm i ` to follow through on their intent
3. Developer gets interrupted or distracted
4. Developer later picks up where they left off and enters `npm i @ungap/structured-clone @types/ungap__structured-clone --save` into the terminal and hits `Enter`, forgetting that `npm i ` was already entered.  The full command `npm i npm i @ungap/structured-clone @types/ungap__structured-clone --save` gets executed.